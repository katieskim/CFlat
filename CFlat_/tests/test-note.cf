int main()
{
	note n;
	n = ( /C-/ /4/ /s./ );
	printn(n);
	
	return 0;
}
