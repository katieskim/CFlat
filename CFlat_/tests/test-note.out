C- 4 s.